qrenderdoc/Windows/EventBrowser.h

```diff
@@ -110,6 +111,22 @@ class ParseErrorTipLabel : public QLabel
   void resizeEvent(QResizeEvent *);
 };
 
+class QRClickToolButton : public QToolButton
```

Comment:
A minor thing, could you move this to `QRDUtils.h` ? That's where I put little controls/overloads like this which don't merit a full separate file implementation. It means if this wants to get used elsewhere in future we don't need to `#include "EventBrowser.h"` to get it.

Reply:
Will do!